src/virtual-scroller.ts

```diff
@@ -628,14 +628,18 @@ export class VirtualScrollerComponent implements OnInit, OnChanges, OnDestroy {
 		let marginLeft = parseInt(styles['margin-left'], 10) || 0;
 		let marginRight = parseInt(styles['margin-right'], 10) || 0;
 
+		/*
+		 * As of typescript 4.5.4 from Angular v13 update this type is missing `x`, `y`, and `toJSON` from the `ClientRect` type.
+		 * Casting as any as to not change the functionality and provide properties that were not there before.
+		 */
 		return {
 			top: result.top + marginTop,
 			bottom: result.bottom + marginBottom,
 			left: result.left + marginLeft,
 			right: result.right + marginRight,
 			width: result.width + marginLeft + marginRight,
 			height: result.height + marginTop + marginBottom
```

Comment:
i would avoid to use anys. i like to respect the type, just to prevent possible errors on the future. for example, if we need the x and y, and we dont use, we can pass as zero. the the toJSON can be done with a simple json stringify. see an exemple in the sugestion
```suggestion
                const result = {
                        x: 0,
                        y: 0,
                        top: result.top + marginTop,
                        bottom: result.bottom + marginBottom,
                        left: result.left + marginLeft,
                        right: result.right + marginRight,
                        width: result.width + marginLeft + marginRight,
                        height: result.height + marginTop + marginBottom,
                      };
                    return {
                      ...base,
                      toJSON: () => {
                        return JSON.stringify(base);
                      }
                    }
```

Reply:
Hi @rintoj is it possible to merge this branch and release a new version?